h/session.py

```diff
@@ -89,6 +89,10 @@ def _group_model(route_url, group):
         model['url'] = route_url('group_read',
                                  pubid=group.pubid,
                                  slug=group.slug)
+
+    if user is not None and user.id == group.creator_id:
```

Comment:
Well, if I'm honest I think pretty much all of the code in `h/session.py` should live in a service that returns a data model, and the view should call a presenter that returns the JSON. Leaving only the `includeme` that sets up the session cookie.

I can start this process by creating a `SessionService` that for now just returns a list of groups for a given user, and in the future we can move more and more of the business logic into that service.